lib/src/constants/strings.dart

```diff
@@ -0,0 +1,11 @@
+class StringData {
+  static const String listCoinsTitle = 'Coins';
+  static const String listEmpty = 'List is empty';
+  static const String hintTextSearch = 'Search';
+  static const String constImage =
+      'https://www.facebook.com/photo/?fbid=2844772675847282&set=a.1403234546667776';
+  static const String errorMessage = 'Failed to load coins list';
+  static const int NUMBER_OF_COINS_PER_STATE = 100;
+  static const String FINAL_API_URL = 'https://api.coingecko.com/api/v3/coins';
```

Comment:
FINAL_API_URL or final_api_url?

Reply:
FINAL_API_URL because it is a convention to define API url that I was learned in college.